modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/rest-config.yaml

```diff
@@ -6,5 +6,5 @@ application:
     name: "Liferay.Headless.Admin.Workflow"
 author: "Javier Gamarra"
 clientDir: "../headless-admin-workflow-client/src/main/java"
-compatibilityVersion: 2
+compatibilityVersion: 3
```

Comment:
I think it is better to update all the `rest-config.yaml` files to 3

Reply:
Okey, I'll do it